app/assets/javascripts/submitFeedback.js

```diff
@@ -0,0 +1,61 @@
+$(document).on('turbolinks:load', function(){
+
+
+  $('.submit_feedback').on('submit', function(event) {
+    var valid = true;
+    var id = $(event.target).attr('id');
+    var form = $(event.target);
+
+    $('select[form="'+ id +'"]').each(function() {
+      if ($(this).val() == '') {
+        valid = false;
+        if (!$(this).hasClass('govuk-input--error')) {
+          $(this).addClass('govuk-input--error');
+          $(this).wrap("<div class='govuk-form-group--error'></div>");
+          text = form.data('optionNotSelectedMessage');
+          $(this).parent('div').prepend('<span class="govuk-error-message">'+ '<span class="govuk-visually-hidden">' + 'Error:' +'</span>' + text + '</span>')
+        }
+      } else {
+        $(this).removeClass('govuk-input--error');
+        $(this).parents('div').children('span').remove();
+        if($(this).parents('div').hasClass('govuk-form-group--error')){
+          $(this).unwrap();
+        }
+      }
+    })
+
+    return valid;
+  })
+
+  $('.submit_feedback').on('ajax:success', function(event) {
+    var form = $(event.target);
+    var notification = $('.notification');
+    var count = Number(notification.text()) - 1;
+    var table = $('table.vacancies')
+    var notice = $('#awaiting_notice')
+    var notice_count = notice.find('.count');
+    var feedback_intro = $('#awaiting_feedback_intro')
+
+    if (count > 0) {
+      row = form.parents('tr');
+      if (count == 1) {
+        notice_count.text('1 job');
+      } else {
+        notice_count.text(count + ' vacancies'); // TODO: Why is this vacancies and where is it appearing?
```

Comment:
Added a third vacancy to the test setup so that when feedback is given for just one, the else condition would be triggered. (The first part of the `if statement` was already covered by tests as of the time of writing).

The associated spec has also be modified to test that `jobs` is displayed rather than `vacancies`. 
The test initially failed since `vacancies` was being displayed. This has now been fixed.

-----------------------------------------------------------------------------------------------

**Before:**
![Screenshot 2019-06-19 at 11 21 55](https://user-images.githubusercontent.com/32823756/59759141-596bfb80-9287-11e9-9aee-e58000cac401.png)

**Results after filling in a vacancy/vacancies:**

1. ![Screenshot 2019-06-19 at 11 22 14](https://user-images.githubusercontent.com/32823756/59759147-5d981900-9287-11e9-8fb5-dc55d06622e9.png)

2. ![Screenshot 2019-06-19 at 11 22 24](https://user-images.githubusercontent.com/32823756/59759466-0d6d8680-9288-11e9-94ad-ec601e0a0347.png)

-----------------------------------------------------------------------------------------------

**After:**
![Screenshot 2019-06-19 at 11 21 55](https://user-images.githubusercontent.com/32823756/59759305-afd93a00-9287-11e9-91ae-120b9f35c030.png)

**Results after filling in a vacancy/vacancies:**

1. ![Screenshot 2019-06-19 at 11 26 09](https://user-images.githubusercontent.com/32823756/59759312-b4055780-9287-11e9-84e5-2cb4ba0600dd.png)

2. ![Screenshot 2019-06-19 at 11 22 24](https://user-images.githubusercontent.com/32823756/59759317-b798de80-9287-11e9-85b2-9e0aff396a96.png)